azure-pipelines-expectation-gallery.yml

```diff
@@ -130,21 +134,100 @@ stages:
 #             done;
             displayName: 'Wait for Trino database to initialise'
 
-#         - task: DownloadSecureFile@1
-#           name: gcp_authkey
-#           displayName: 'Download Google Service Account'
-#           inputs:
-#             secureFile: 'superconductive-service-acct_ge-oss-ci-cd.json'
-#             retryCount: '2'
+          - task: DownloadSecureFile@1
+            name: gcp_authkey
+            displayName: 'Download Google Service Account'
+            inputs:
+              secureFile: 'superconductive-service-acct_ge-oss-ci-cd.json'
+              retryCount: '2'
+
+          - task: DownloadSecureFile@1
+            name: aws_authkey
+            displayName: 'Download AWS Credentials'
+            inputs:
+              secureFile: 'aws_config_sandbox'
+              retryCount: '2'
 
-          - bash: python ./build_gallery.py
+          - bash: python ./build_gallery.py --outfile-name "expectation_library_v3.json" | tee output--build_gallery.txt ; grep -o "Took .* seconds to .*" output--build_gallery.txt | sort -k2,2nr > testing-times.txt ; grep -o "ERROR - (.*" output--build_gallery.txt | sort > testing-error-messages.txt; touch gallery-tracebacks.txt; grep -o "Expectation type.*" output--build_gallery.txt | sort > gallery-exp-types.txt
```

Comment:
much respec :) I really do like the validation here.

Reply:
Yea it powers all those little summary/info sections below